src/lib_ccaligner/grammar_tools.cpp

```diff
@@ -17,6 +17,92 @@ static int systemGetStatus(const char* command) {
 #endif
 }
 
+void InitializeGenerator(bool &generateQuickDict, bool &generateQuickLM, grammarName &name)
```

Comment:
Please rename it to something more appropriate like `ConfigureQuickGenerationOptions`.